django_tables2/columns/templatecolumn.py

```diff
@@ -56,11 +67,18 @@ def render(self, record, table, value, bound_column, **kwargs):
         additional_context = {
             "default": bound_column.default,
             "column": bound_column,
-            "record": record,
+            self.context_object_name: record,
             "value": value,
             "row_counter": kwargs["bound_row"].row_counter,
         }
-        additional_context.update(self.extra_context)
+
+        extra_context = self.extra_context
```

Comment:
ik vind "additional" en "extra" content wel dubbelop? Waar is additional voor en waar is extra voor? Kunnen we de namen niet wat aanpassen in termen van waar ze voor dienen?